### ISSUE TITLE
DuplicateProperty vendor-prefixed values code doesn't handle lists

### ISSUE THREAD

**Issue by srawlins** [brigade/scss-lint#737]
Take the following SCSS:

```scss
$imp: '!important';

p {
  display: -moz-box unquote($imp);
  margin: 2px;
  display: -ms-flexbox unquote($imp);
}
```

DuplicateProperty should key the two `display` lines separately, because the values start with `-` (vendor prefixes), not counting them as duplicates, but fails:

```
$ bundle exec bin/scss-lint x.scss
x.scss:6 [W] DuplicateProperty: Property `display` already defined on line 4
```
